api/model/src/main/java/org/projectnessie/error/NessieError.java

```diff
@@ -111,13 +109,10 @@ default String getFullMessage() {
       sb.append("\n").append(getServerStackTrace());
     }
 
-    if (getClientProcessingException() != null) {
-      StringWriter sw =
-          new StringWriter()
-              .append(
-                  "\nAdditionally, the client-side exception below was caught while decoding the HTTP response:\n");
-      getClientProcessingException().printStackTrace(new PrintWriter(sw));
-      sb.append("\n").append(sw);
+    if (getClientProcessingError() != null) {
+      sb.append(
+              "\nAdditionally, the client-side exception below was caught while decoding the HTTP response: ")
```

Comment:
nit: `Additionally, a client-side error was detected while decoding the HTTP response: ` (it may not be limited to exceptions now)